modules/pre-commit.nix

```diff
@@ -174,7 +174,7 @@ let
       chmod -R +w src
       ln -fs ${configFile} src/.pre-commit-config.yaml
       cd src
-      rm -rf src/.git
+      rm -rf .git
```

Comment:
```suggestion
      if [[ -e .git ]]; then
        echo 'error: The src of your pre-commit check contains a .git directory. Please filter it out, so that the check is more reproducible and more efficient with store resources.'
        exit 1
      fi
```